docs/wiki/configurator/failsafe-tab.md

```diff
@@ -4,8 +4,125 @@ sidebar_position: 5
+
 :::note
-Sorry; this Wiki page has not yet been written.
+Values are saved in steps of 25usec, so small changes disappear
 :::
 
-For more information about Failsafe, see the [Wiki document](/docs/wiki/guides/current/GPS-Rescue-v4-5).
+You can set the channel fallback settings for each channel individually.
+
+The available options for each channel are:
+
+- **Auto** - Roll, Pitch and Yaw will be set to the center position, Throttle will be set to the minimum value.
+  (this is only available for the first 4 channels)
+- **Hold** - The channel will hold its last valid value.
+- **Set** - You can set a custom value for the channel.
+
+## Stage 2 - Failsafe Procedure
+
+There are three options for the failsafe procedure:
+
+- **Drop** (default) - The craft will disarm and fall out of the sky.
+- **Land** - The craft will attempt to land.
+- **GPS Rescue** - The craft will attempt to return to the home position.
+
+### Land options
+
+- **Throttle value used while landing** - The throttle value the craft will use while landing.
+- **Delay for turning off the motors during Failsafe [seconds]** - Time to stay in landing mode until the motors turn off.
+
+:::warning
+Betaflight can't detect when the craft has landed, it will only disarm after the delay has passed.
+Use this feature with caution.
+
+If the throttle value is too high, the craft will ascend instead of landing.
+:::
+
+### GPS Rescue options
+
+Here’s the revised table with the sublists formatted in one row:
+
+| **Option**                              | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Altitude mode**                       | <li>**Maximum Altitude** - The craft will ascend to the maximum altitude during that flight plus `GPS_RESCUE_INITAL_CLIMB`.</li><li>**Fixed Altitude** - The craft will ascend to a fixed altitude.</li><li>**Current Altitude** - The craft will fly back at the altitude it was at when the failsafe occurred.</li>                                                                                                                             |
```

Comment:
**Fixed Altitude** - The craft will ascend to the configured Return Altitude.
**Current Altitude** - The craft will fly back at the altitude it was at when the failsafe occurred, plus the Initial Climb altitude, or at the Initial Climb Altitude if the failsafe occurred below the Home Altitude .